sample-app/src/components/SearchBar.tsx

```diff
@@ -59,13 +59,16 @@ export default function SearchBar({ placeholder, isVertical, screenReaderInstruc
         placeholder={placeholder}
         screenReaderInstructions={SCREENREADER_INSTRUCTIONS}
         screenReaderInstructionsId={screenReaderInstructionsId}
-        options={autocompleteResults.map(result => {
-          return {
-            value: result.value,
-            render: () => renderWithHighlighting(result)
-          }
-        })}
+        options={autocompleteResults.length > 0 ? [{
+          results: autocompleteResults.map(result => {
+            return {
+              value: result.value,
+              render: () => renderWithHighlighting(result)
+            }
+          })
+        }] : []}
```

Comment:
can we simplify to without the ternary operator?

Reply:
I added the ternary operator because without it, it would pass `[{ results: [] }]` as `options`, which results in `options.length > 0` checks incorrectly passing

Reply:
I see that filterSearch, on a no results would have `sections: [{results: []}]` in the network response, so the filterSearch component right now would have `[{ results: [] }]` options. That's working?

Reply:
When there are no results found, executeFilterSearch returns `sections: []` so it passes `options: []` to the InputDropdown